src/Service/MercadopagoManager.php

```diff
@@ -33,4 +33,63 @@ public function configure()
     {
         MercadoPago\SDK::setAccessToken($this->settingsManager->get('mercadopago_access_token'));
     }
+
+    /**
+     * @return MercadoPago\Payment
+     */
+    public function authorize(PaymentInterface $payment)
+    {
+        $this->configure();
+
+        $order = $payment->getOrder();
+
+        $options = [];
+
+        $applicationFee = 0;
+        if (null !== $order->getRestaurant()) {
+            $account = $order->getRestaurant()->getMercadopagoAccount(false);
+            if ($account) {
+                $applicationFee = $order->getFeeTotal();
+                // @see MercadoPago\Manager::processOptions()
+                $options['custom_access_token'] = $account->getAccessToken();
+            }
+        }
+
+        $order = $payment->getOrder();
+
+        $p = new MercadoPago\Payment();
+
+        $p->transaction_amount = ($payment->getAmount() / 100);
+        $p->token = $payment->getStripeToken();
+        $p->description = sprintf('Order %s', $order->getNumber());
+        $p->installments = $payment->getMercadopagoInstallments() ?? 1;
+        $p->payment_method_id = $payment->getMercadopagoPaymentMethod();
+        $p->payer = array(
+            'email' => $order->getCustomer()->getEmail()
+        );
+        $p->capture = false;
+
+        if ($applicationFee > 0) {
+            $p->application_fee = ($applicationFee / 100);
+        }
+
+        $p->save($options);
+
+        return $p;
+    }
+
+    /**
+     * @return MercadoPago\Payment
+     */
+    public function capture(PaymentInterface $payment)
+    {
+        $this->configure();
+
+        $payment = MercadoPago\Payment::find_by_id($payment->getCharge());
+        $payment->capture = true;
+
+        $payment->update();
```

Comment:
We _probably_ need to add an `$options` parameter with the merchant access token, like in the `authorize()` method. I think it needs mercadopago/dx-php#273

Reply:
Fixed in 57421acc8d38469dfe93bb3b9db4cbad2a14af20